frontend/src/components/program/picasso/PicassoEntry.vue

```diff
@@ -265,7 +265,7 @@ export default {
     this.scrollHeight = this.$el.scrollHeight
     window.addEventListener('resize', this.onResize)
   },
-  destroyed() {
+  unmounted() {
```

Comment:
This one got forgotten to revert.

```suggestion
  destroyed() {
```